webpack.config.js

```diff
@@ -14,15 +13,44 @@ const WebpackBar = require( 'webpackbar' );
  */
 const defaultConfig = require( '@wordpress/scripts/config/webpack.config' );
 const DependencyExtractionWebpackPlugin = require( '@wordpress/dependency-extraction-webpack-plugin' );
-const { defaultRequestToExternal, defaultRequestToHandle, camelCaseDash } = require( '@wordpress/dependency-extraction-webpack-plugin/util' );
+const { defaultRequestToExternal, defaultRequestToHandle, camelCaseDash } = require( '@wordpress/dependency-extraction-webpack-plugin/lib/util' );
 
 const sharedConfig = {
+	...defaultConfig,
 	output: {
 		path: path.resolve( process.cwd(), 'assets', 'js' ),
 		filename: '[name].js',
 		chunkFilename: '[name].js',
 	},
+	module: {
+		...defaultConfig.module,
+		rules: [
+			...defaultConfig.module.rules.map(
+				( rule ) => {
+					// @todo Can remove once the default config no longer excludes excludes CSS related to Gutenberg components.
```

Comment:
The `setup` Webpack config fails when implicitly importing a CSS file from `node_modules/@wordpress/components/build-style/style.css`. I've modified the rule to include the `node_modules/@wordpress` directory for now, until the issue is resolved upstream.

I've opened https://github.com/WordPress/gutenberg/issues/23402 to track this.